Civi/Core/SettingsBag.php

```diff
@@ -182,6 +182,9 @@ public function all() {
    */
   public function get($key) {
     $all = $this->all();
+    if ($key === 'contribution_invoice_settings') {
+      \CRM_Core_Error::deprecatedWarning('Invoicing settings should be directly accessed - eg Civi::setting()->get("invoicing")');
```

Comment:
This is a typo I've made myself 18 times - should be Civi::settings not Civi::setting.